cernopendata/modules/fixtures/data/records/jade-logbooks.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "abstract": {
-      "description": "This is JADE logbook 01 from March 10th 1979 to August 9th 1979 with runs 1 to 1337 described on 189 pages. These logbooks are a handwritten collection of incidents in the format of a timeline and a list of properties for most of the Runs done with the PETRA accelerator. This specific one contains a number of descriptions of computer errors, near overheating of the record tapes and the first signs of three tracks."
+      "description": "This is JADE logbook 01 from March 10th 1979 to August 9th 1979 data taking with runs 1 to 1337 described on 189 pages. These logbooks contain handwritten notes of the JADE online data taking shift crews, logging the status and conditions of the PETRA accelerator and beams, of the JADE detector, of the data taking system, the general properties of all recorded data runs and of actual incidents and decisions concerning the operation of PETRA and of JADE. \n\nThis specific one contains a number of descriptions of computer errors, failures of electronics components, and notes the fatal beam loss on March 26 which severly damaged the central tracking (“Jet”) chamber, causing a 3-months shutdown and repair of JADE while PETRA and the other experiments (MARK-J, PLUTO and TASSO)  continued data taking. These early data provided evidence for 3-jet events and thus for the discovery of the gluon, and found no evidence for the top-quark at collision energies up to 32 GeV."
```

Comment:
Thanks, I've replaced newlines with `<p>...</p>` to have paragraph separation. (The HTML markup is allowed in record descriptions, the newlines characters would not turn visible.)

Reply:
Thanks!